orhan.js

```diff
@@ -0,0 +1,102 @@
+//1. Part of Homework
+
+console.log('Merhaba, Dunya!\nHello, World!\nHallo, Wereld!');
+
+//2. Part of Homework
+
+console.log('I\'m awesome');
+//this also works console.log("I'm awesome");
+
+//3. Part of Homework
+
+let x;
+console.log('the value of my variable x will be: undefined');
+console.log(x);
+x = 10;
+console.log('the value of my variable x will be: 10');
+console.log(x);
+
+//4. Part of Homework
+
+let y = 'hello';
+console.log('the value of my variable y will be: hello');
+console.log(y);
+y = 'hi';
+console.log('the value of my variable y will be: hi');
+console.log(y);
+
+//5. Part of Homework
+
+let z = 7.25;
+console.log('the value of z is: ' + z);
+let a = Math.round(z);
+console.log('the value of a is: ' + a);
+let highestValue = a < z ? z : a;
+console.log('the highest value is: ' + highestValue);
```

Comment:
Two things here please:
1. You can use `const` keyword instead of `let` to create `z`, `a` and `highestValue` as we are not re-assigning any of those variables, right?
2. I would like you to learn about the difference between `Math.floor` and `Math.round` methods 🙂